vms/platformvm/utxo/camino_handler_test.go

```diff
@@ -106,3 +109,422 @@ func TestVerifySpendUTXOsWithLocked(t *testing.T) {
+					Amt: 1,
+					OutputOwners: secp256k1fx.OutputOwners{
+						Addrs: []ids.ShortID{
+							addresses[0],
+						}, Threshold: 1,
+					},
+				},
+			},
+			producedAmounts: map[ids.ID]uint64{
+				h.ctx.AVAXAssetID: 1,
+			},
+			expectedErr: ErrWrongUTXONumber,
+		},
+		"Zero Signers, Threshold=1": {
+			utxos: []*avax.UTXO{
+				{
+					Asset: avax.Asset{ID: h.ctx.AVAXAssetID},
+					Out: &secp256k1fx.TransferOutput{
+						Amt: 1,
+					},
+				},
+			},
+			ins: []*avax.TransferableInput{
+				{
+					Asset: avax.Asset{ID: h.ctx.AVAXAssetID},
+					In: &secp256k1fx.TransferInput{
+						Amt: 1,
+					},
+				},
+			},
+			outs: []*avax.TransferableOutput{
+				{
+					Asset: avax.Asset{ID: h.ctx.AVAXAssetID},
+					Out: &secp256k1fx.TransferOutput{
+						Amt: 1,
+					},
+				},
+			},
+			creds: []verify.Verifiable{
+				&secp256k1fx.Credential{},
+			},
+			signers: []verify.State{
+				&secp256k1fx.TransferOutput{
+					Amt: 1,
+					OutputOwners: secp256k1fx.OutputOwners{
+						Addrs: []ids.ShortID{
+							addresses[0],
+						}, Threshold: 1,
+					},
+				},
+			},
+			producedAmounts: map[ids.ID]uint64{},
+			expectedErr:     secp256k1fx.ErrTooFewSigners,
+		},
+		"Two owners, Threshold=1, one signature": {
+			utxos: []*avax.UTXO{
+				{
+					Asset: avax.Asset{ID: h.ctx.AVAXAssetID},
+					Out: &secp256k1fx.TransferOutput{
+						Amt: 1,
+					},
+				},
+			},
+			ins: []*avax.TransferableInput{
+				{
+					Asset: avax.Asset{ID: h.ctx.AVAXAssetID},
+					In: &secp256k1fx.TransferInput{
+						Amt: 1,
+						Input: secp256k1fx.Input{
+							SigIndices: []uint32{1},
+						},
+					},
+				},
+			},
+			outs: []*avax.TransferableOutput{
+				{
+					Asset: avax.Asset{ID: h.ctx.AVAXAssetID},
+					Out: &secp256k1fx.TransferOutput{
+						Amt: 1,
+					},
+				},
+			},
+			creds: []verify.Verifiable{
+				&secp256k1fx.Credential{
+					Sigs: [][crypto.SECP256K1RSigLen]byte{
+						credentials[1].Sigs[0],
+					},
+				},
+			},
+			signers: []verify.State{
+				&secp256k1fx.TransferOutput{
+					Amt: 1,
+					OutputOwners: secp256k1fx.OutputOwners{
+						Addrs: []ids.ShortID{
+							addresses[0],
+							addresses[1],
+						}, Threshold: 1,
+					},
+				},
+			},
+			producedAmounts: map[ids.ID]uint64{},
+			expectedErr:     nil,
+		},
+		"More signatures than is enough cause the error": {
+			utxos: []*avax.UTXO{
+				{
+					Asset: avax.Asset{ID: h.ctx.AVAXAssetID},
+					Out: &secp256k1fx.TransferOutput{
+						Amt: 1,
+					},
+				},
+			},
+			ins: []*avax.TransferableInput{
+				{
+					Asset: avax.Asset{ID: h.ctx.AVAXAssetID},
+					In: &secp256k1fx.TransferInput{
+						Amt: 1,
+						Input: secp256k1fx.Input{
+							SigIndices: []uint32{0, 1},
+						},
+					},
+				},
+			},
+			outs: []*avax.TransferableOutput{
+				{
+					Asset: avax.Asset{ID: h.ctx.AVAXAssetID},
+					Out: &secp256k1fx.TransferOutput{
+						Amt: 1,
+					},
+				},
+			},
+			creds: []verify.Verifiable{
```

Comment:
Here add one more credential.
(it is unlikely sigIndices and credentials are unmatched)